docs/gmrq.rst

```diff
@@ -1,8 +1,33 @@
 .. _gmrq:
 
-Model Selection (GMRQ)
+Model Selection using GMRQ
```

Comment:
I guess. I just don't think GMRQ is the end-all-be-all for this. Either this section is about GMRQ or the broader topic of model selection, which can be expanded as more methods are developed.